CONTRIBUTING.md

```diff
@@ -0,0 +1,118 @@
+# CONTRIBUTION GUIDELINES
+Welcome to the Hyperledger Labs AIFAQ project! We're excited to have you contribute and help us improve our project. Below are the guidelines to ensure a smooth and efficient contribution process.
+
+## Code of Conduct
+Please adhere to our [Code of Conduct](https://wiki.hyperledger.org/pages/viewpage.action?pageId=41587043). This code of conduct outlines our expectations for participants within the community as well as steps to report unacceptable behavior.
+
+## Getting Started
+
+### Frontend 
+First, run the development server:
+
+1.Clone the repository:
+```bash 
+git clone https://github.com/hyperledger-labs/aifaq.git
+cd aifaq/frontend
+```
+
+2.Install dependencies:
+```bash
+yarn install
+```
+
+3.Start the developemnt server:
+```bash
+yarn dev
+```
+
+4.Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
+
+### Backend
+Run the AIFAQ API through [lighning.ai](https://lightning.ai/studios) by following this [short guide](https://docs.google.com/document/d/1NM3EOmKGdaSS476pWRGnGdJMZpL9DMO4_nZlu6s3Myw/edit)
```

Comment:
Instead of this please add instructions for running locally.

Reply:
Alright @swaptr .I have added the instructions PTAL